index.test.ts

```diff
@@ -375,4 +375,35 @@ describe('Sort JSON', () => {
 
     expect(output).toMatchSnapshot();
   });
+
+  it('should sort JSON objects within an array', () => {
```

Comment:
To differentiate this from the other test of sorting JSON objects in arrays, I'd suggest this description:

```suggestion
  it('should sort JSON objects recursively within a nested array', () => {
```

or maybe

```suggestion
  it('should sort JSON objects recursively within a non-top-level array', () => {
```

The word "recursively" was added despite that being self-evident as a hint that this pertains to the `jsonRecursiveSort` option.

Reply:
Committed suggestion.